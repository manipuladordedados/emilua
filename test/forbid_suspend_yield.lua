this_fiber.forbid_suspend()
this_fiber.yield()
