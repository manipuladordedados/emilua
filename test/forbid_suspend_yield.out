Main fiber from VM 0x1 panicked: 'EPERM within a forbid-suspend block'
stack traceback:
	input.lua:2: in main chunk
	[C]: in function ''
	[string "?"]: in function <[string "?"]:0>
